docs/JestObjectAPI.md

```diff
@@ -314,6 +314,46 @@ test('moduleName 2', () => {
 });
 ```
 
+Using `jest.doMock()` with ES6 modules requires additional steps:
+
+- We have to specify the `__esModule: true` property (see the [`jest.mock()`](#jestmockmodulename-factory-options) API for more information).
+- Static ES6 module imports can only appear at the top of the file, so instead we have to import them dynamically using `import()`.
```

Comment:
This is not actually correct, they can appear anywhere, but are effectively hoisted to the top

Reply:
Ah thanks for letting me know. I actually got a syntax error when trying it, but I guess it must have been something with Babel.

Changed.